### DISCUSSION TITLE
Both Octal and RWX

### DISCUSSION THREAD

**Discussion by julas23** [lsd-rs/lsd#895] [Ideas]
I am a Linux Native Heavy User since 1994 (Debian 0.9), and I'd like to suggest a feature please.

I realized that recent users on Linux, see difficulties on Permissions, RWX is easier to understand, but Octal is a little bit harder, so, could keep both, instead choose one of them, to help and provide information to adapt new comers to Linux. And if possible in the header keep something like this:

Permission (r=4, w=2, x=1)
0644     .rw-r--r--  juliano juliano 5.6 KB Tue Sep  5 18:04:55 2023  .bashrc

**Comment by Kol9yN** [comment]
Agreed! EXA, and EZA already have it! Should be here too i think
Vote about issue here please
https://github.com/lsd-rs/lsd/issues/986
